Delegate_CategorySampleApp/ViewController.m

```diff
@@ -0,0 +1,78 @@
+//
+//  ViewController.m
+//  Delegate_CategorySampleApp
+//
+//  Created by webonise on 21/09/16.
+//  Copyright © 2016 webonise. All rights reserved.
+//
+
+#import "ViewController.h"
+
+@interface ViewController ()
+
+@end
+
+@implementation ViewController
+CustomAlertView *customAlertView;
+UIView *view1;
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view, typically from a nib.
+    customAlertView = [[CustomAlertView alloc]init];
+    customAlertView.delegate = self;
+    [customAlertView designAlertView :@"Do you want to delete?" :@"cancel" :@"OK"];
```

Comment:
- create initializer with parameters for message, firstButtonTitle and secondButtonTitle, delegate ,
- as per current implementation you did not take advantage of it super class UIView.